tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
```

Comment:
*If* the only reason we're including some tests is to demonstrate one of the args to `exclude=`, I'd actually much rather show people a basic `docs` example to get started with sphinx. Advantage there is that it's already established as single popular way to do docs for python.